### ISSUE TITLE
Validation Loss while training the InternVL2.

### ISSUE THREAD

**Issue by KaranBhuva22** [OpenGVLab/InternVL#451]
### Checklist

- [X] 1. I have searched related issues but cannot get the expected help.
- [X] 2. The bug has not been fixed in the latest version.
- [X] 3. Please note that if the bug-related issue you submitted lacks corresponding environment info and a minimal reproducible demo, it will be challenging for us to reproduce and resolve the issue, reducing the likelihood of receiving feedback.

### Describe the bug

@czczup  'm trying to determine if loss and accuracy metrics were computed during training. After reviewing the code, I couldn't locate this information. Can you please provide guidance on incorporating validation metrics into the training process?

### Reproduction

.

### Environment

```Shell
.
```

### Error traceback

```Shell
.
```

**Comment by czczup** [comment]
Hello, during our training process, we did not use a validation set nor did we monitor the loss on the validation set. We primarily focused on the loss on the training set.

**Comment by KaranBhuva22** [comment]
Hey @czczup, thanks for your answer. If I want to fine-tune Intenvl for a specific task, how can I determine if the training is successful without a validation set? Do I need to adjust the code to add a validation set, or is it enough to evaluate the model on the training data alone?
